backend/src/v4/routes/auth.js

```diff
@@ -25,125 +25,15 @@ const C = require("../constants");
 const middlewares = require("../middlewares/middlewares");
```

Comment:
yea I don't particularly care and I think we'll just delete the whole folder when we're ready (Also worried that there might be weird dependencies around the model files that would cause unexpected bugs... it's v4, anything goes 😢 )

The intention is really to just unregister the routes so it can't be accessed externally.

Reply:
Hitting resolve on this, unresolve and I'll take a look if it bothers you 😆